db/blip_sync_context.go

```diff
@@ -376,7 +376,10 @@ func (bsc *BlipSyncContext) handleChangesResponse(ctx context.Context, sender *b
 			if err != nil {
 				return err
 			}
-
+			base.Audit(ctx, base.AuditIDDocumentRead, base.AuditFields{
```

Comment:
This looks like a duplicate call with the event in sendRevisionWithProperties, maybe just needs cleanup?